scripts/spack/packages/axomdevtools/package.py

```diff
@@ -12,12 +12,18 @@ class Axomdevtools(BundlePackage):
 
     maintainers = ['white238']
 
-    depends_on("python")
+    depends_on("python@3.11")
     depends_on("doxygen")
     depends_on("cppcheck+rules+htmlreport")
     depends_on("graphviz")
     depends_on("py-sphinx")
     depends_on("py-shroud")
     depends_on("py-sphinxcontrib-jquery")
-    depends_on("py-jsonschema")
-    depends_on("llvm+clang@10.0.0")
+
+    # 4.18 builds py-rpds-py, which then needs rust...
+    depends_on("py-jsonschema@4.17")
+    depends_on("py-nanobind@2.7.0")
+    depends_on("py-pytest")
+    depends_on("py-numpy")
+
+    depends_on("llvm+clang@14")
```

Comment:
As discussed previously, devtools modified to explicitly requires llvm+clang@14 for clang-format.